ot-node.js

```diff
@@ -163,15 +163,7 @@ class OTNode {
             .getImplementationNames()
             .map(async (blockchain) => {
                 try {
-                    if (!blockchainModuleManager.identityExists(blockchain)) {
-                        this.logger.info(`Creating blockchain identity on network: ${blockchain}`);
-                        await blockchainModuleManager.deployIdentity(blockchain);
-                        await blockchainModuleManager.saveIdentityInFile();
-                    }
-                    const identity = blockchainModuleManager.getIdentity(blockchain);
-                    this.logger.info(`${blockchain} blockchain identity is ${identity}`);
-
-                    if (!(await blockchainModuleManager.profileExists(blockchain, identity))) {
+                    if (!(await blockchainModuleManager.profileExists(blockchain))) {
```

Comment:
It looks to me that we are missing identity creation now, where it will be created?

Reply:
I see now in PR description that it will be created during profile creation, don't mind this comment